### PR TITLE
Modified support-matrix links

### PR DESCRIPTION
**Pull request by serenashe**
Signed-off-by: adakisme <serenashe@yunify.com>

### CODE REVIEW

File: content/en/docs/installing-on-linux/introduction/air-gapped-installation.md [hunk 1]
```diff
@@ -204,7 +204,7 @@ As you install KubeSphere and Kubernetes on Linux, you need to prepare an image
 
    {{< notice note >}}
 
-   - You can change the Kubernetes version downloaded based on your needs. Recommended Kubernetes versions for KubeSphere v3.1.0: v1.17.9, v1.18.8, v1.19.8 and v1.20.4. If you do not specify a Kubernetes version, KubeKey will install Kubernetes v1.19.8 by default. For more information about supported Kubernetes versions, see [Support Matrix](/docs/installing-on-linux/introduction/kubekey/#support-matrix).
+   - You can change the Kubernetes version downloaded based on your needs. Recommended Kubernetes versions for KubeSphere v3.1.0: v1.17.9, v1.18.8, v1.19.8 and v1.20.4. If you do not specify a Kubernetes version, KubeKey will install Kubernetes v1.19.8 by default. For more information about supported Kubernetes versions, see [Support Matrix](../../../installing-on-linux/introduction/kubekey/#support-matrix).
```
Comment (Sherlock113) [inline content/en/docs/installing-on-linux/introduction/air-gapped-installation.md:207]: ->
../kubekey/#support-matrix

File: content/en/docs/installing-on-linux/introduction/multioverview.md [hunk 2]
```diff
@@ -151,7 +151,7 @@ Command:
 
 {{< notice note >}}
 
-- Recommended Kubernetes versions for KubeSphere v3.1.0: v1.17.9, v1.18.8, v1.19.8 and v1.20.4. If you do not specify a Kubernetes version, KubeKey will install Kubernetes v1.19.8 by default. For more information about supported Kubernetes versions, see [Support Matrix](/docs/installing-on-linux/introduction/kubekey/#support-matrix).
+- Recommended Kubernetes versions for KubeSphere v3.1.0: v1.17.9, v1.18.8, v1.19.8 and v1.20.4. If you do not specify a Kubernetes version, KubeKey will install Kubernetes v1.19.8 by default. For more information about supported Kubernetes versions, see [Support Matrix](../../../installing-on-linux/introduction/kubekey/#support-matrix).
```
Comment (Sherlock113) [inline content/en/docs/installing-on-linux/introduction/multioverview.md:154]: ->
../kubekey/#support-matrix

File: content/en/docs/quick-start/all-in-one-on-linux.md [hunk 3]
```diff
@@ -141,7 +141,7 @@ To create a Kubernetes cluster with KubeSphere installed, refer to the following
 
 {{< notice note >}}
 
-- Recommended Kubernetes versions for KubeSphere v3.1.0: v1.17.9, v1.18.8, v1.19.8 and v1.20.4. If you do not specify a Kubernetes version, KubeKey will install Kubernetes v1.19.8 by default. For more information about supported Kubernetes versions, see [Support Matrix](/docs/installing-on-linux/introduction/kubekey/#support-matrix).
+- Recommended Kubernetes versions for KubeSphere v3.1.0: v1.17.9, v1.18.8, v1.19.8 and v1.20.4. If you do not specify a Kubernetes version, KubeKey will install Kubernetes v1.19.8 by default. For more information about supported Kubernetes versions, see [Support Matrix](../../../installing-on-linux/introduction/kubekey/#support-matrix).
```
Comment (Sherlock113) [inline content/en/docs/quick-start/all-in-one-on-linux.md:144]: This is a dead link.

File: content/en/docs/upgrade/Air-Gapped-Upgrade-with-KubeKey.md [hunk 4]
```diff
@@ -121,7 +121,7 @@ As you install KubeSphere and Kubernetes on Linux, you need to prepare an image
 
    {{< notice note >}}
 
-   - You can change the Kubernetes version downloaded based on your needs. Recommended Kubernetes versions for KubeSphere v3.1.0: v1.17.9, v1.18.8, v1.19.8 and v1.20.4. If you do not specify a Kubernetes version, KubeKey will install Kubernetes v1.19.8 by default. For more information about supported Kubernetes versions, see [Support Matrix](/docs/installing-on-linux/introduction/kubekey/#support-matrix).
+   - You can change the Kubernetes version downloaded based on your needs. Recommended Kubernetes versions for KubeSphere v3.1.0: v1.17.9, v1.18.8, v1.19.8 and v1.20.4. If you do not specify a Kubernetes version, KubeKey will install Kubernetes v1.19.8 by default. For more information about supported Kubernetes versions, see [Support Matrix](../../../installing-on-linux/introduction/kubekey/#support-matrix).
```
Comment (Sherlock113) [inline content/en/docs/upgrade/Air-Gapped-Upgrade-with-KubeKey.md:124]: Dead link.

Comment (serenashe) [reply content/en/docs/upgrade/Air-Gapped-Upgrade-with-KubeKey.md]: Fixed. Thanks so much.

**Comment by Sherlock113** [comment]
@serenashe You need to change the Chinese links as well.
Click every link in this pr before you submit it.

**Comment by serenashe** [comment]
> @serenashe You need to change the Chinese links as well.
> Click every link in this pr before you submit it.

I checked every link in this PR and they all work now. Please take a look, many thanks.

**Pull request merged**